File: PHP/Baju.php
<?php
require_once("Aksesoris.php");
class Baju extends Aksesoris {
  private $untuk;
  private $size;
  private $merk;

  public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna, $untuk, $size, $merk) {
    parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
    $this->untuk = $untuk;
    $this->size = $size;
    $this->merk = $merk;
  }

  public function getUntuk() {
    return $this->untuk;
  }

  public function getSize() {
    return $this->size;
  }

  public function getMerk() {
    return $this->merk;
  }

  public function setUntuk($untuk) {
    $this->untuk = $untuk;
  }

  public function setSize($size) {
    $this->size = $size;
  }

  public function setMerk($merk) {
    $this->merk = $merk;
  }

  public function print() {
    parent::print();
    echo "Untuk: " . $this->untuk . "<br>";
    echo "Size: " . $this->size . "<br>";
    echo "Merk: " . $this->merk . "<br>";
  }
}







?>

File: Python/Baju.py
from Aksesoris import Aksesoris
class Baju(Aksesoris):
  def __init__(self,
    id=None, 
    name="", 
    category="", 
    price=0, 
    size=0, 
    jenis="", 
    warna="", 
    bahan="", 
    untuk="", 
    merk=""):
    super().__init__(id, name, category, price, jenis, warna, bahan)
    self._size = size
    self._untuk = untuk
    self._merk = merk

  def set_size(self, size):
    self._size = size

  def get_size(self):
    return self._size
  
  def set_untuk(self, untuk):
    self._untuk = untuk

  def get_untuk(self):
    return self._untuk
  
  def set_merk(self, merk):
    self._merk = merk

  def get_merk(self):
    return self._merk
  
  def show_baju(self):
    print(f"{self._id:<5}{self._name:<15}{self._category:<10}Rp {self._price:<12}{self._size:<5}{self._jenis:<15}{self._warna:<10}{self._bahan:<10}{self._untuk:<10}{self._merk:<10}")

  
